win32ss/user/ntuser/menu.c

```diff
@@ -2803,12 +2803,11 @@ static BOOL MENU_InitPopup( PWND pWndOwner, PMENU menu, UINT flags )
     if (flags & TPM_LAYOUTRTL || pWndOwner->ExStyle & WS_EX_LAYOUTRTL)
        ex_style = WS_EX_LAYOUTRTL;
 
-    ClassName.Buffer = WC_MENU;
-    ClassName.Length = 0;
+    RtlInitUnicodeString(&ClassName, WC_MENU);
 
     RtlZeroMemory(&WindowName, sizeof(WindowName));
     RtlZeroMemory(&Cs, sizeof(Cs));
-    Cs.style = WS_POPUP;
+    Cs.style = WS_POPUP | WS_CLIPSIBLINGS | WS_BORDER;
```

Comment:
Does this change the visual appearance of the menu?

Reply:
This is `WS_POPUP`:
![WS_POPUP](https://user-images.githubusercontent.com/2107452/62707455-5641f000-ba2c-11e9-9f37-3225b712689e.png)

This is `WS_POPUP + WS_BORDER`:
![WS_POPUP+WS_BORDER](https://user-images.githubusercontent.com/2107452/62707456-5641f000-ba2c-11e9-888b-0452b84b700f.png)

This is `WS_POPUP + WS_BORDER` and `WS_EX_DLGMODALFRAME`:
![WS_POPUP+WS_BORDER+WS_EX_DLGMODALFRAME](https://user-images.githubusercontent.com/2107452/62707453-5641f000-ba2c-11e9-9a6e-0446a5917d80.png)

Reply:
In addition, `CS_DROPSHADOW` will add a shadow to the window.